### PR TITLE
Read both envvars / static error page

### PR DESCRIPTION
**Pull request by mccalluc**

### CODE REVIEW

File: on_startup.py [hunk 1]
```diff
@@ -135,5 +144,28 @@ def main():
...
-    main()
+    try:
+        main()
+    except Exception as e:
+        error_str = ''.join(
+            traceback.TracebackException.from_exception(e).format()
+        )
+        warn(error_str)
+
+        html = '''
```
Comment (scottx611x) [inline on_startup.py]: Awesome! Thanks for doing this!

**Review by scottx611x — APPROVED** [review]
👍

**Review by scottx611x — CHANGES_REQUESTED** [review]
Just saw the failing build, temp revert of approval

File: on_startup.py [hunk 2]
```diff
@@ -135,5 +144,28 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    try:
+        main()
+    except Exception as e:
```
Comment (scottx611x) [inline on_startup.py]: Failing test seems to be due to the fact  that we're catching something with this except that we weren't before

I remember there being something off with the `django.setup()` within main. Re: [this TODO](https://github.com/refinery-platform/refinery-higlass-docker/pull/15/files#diff-9dfe42fae1cc52c3fbaa4df18f8b4d5aR92).

 Maybe try pulling that out as a call before main and see what happens?

**Comment by mccalluc** [comment]
Scott and I talked: the error isn't new, but it wasn't being caught this way before. Think it should pass on Travis now, but the underlying issue should not be forgotten.

**Pull request merged**